include/capstone/riscv_insn.gen.inc

```diff
@@ -0,0 +1,984 @@
+/*=======================================================================*/
+/*This code was generated by the tool riscv_disasm_from_sail (see
+ * https://github.com/moste00/riscv_disasm_from_sail)*/
```

Comment:
1) Should rename the project, maybe since it's capstone auto-sync, something like `capstone-autosync-sail`?
2) Once the repository transferred to rizinorg, also update the URL

cc @wargio